Mathlib/CategoryTheory/Sites/Sheafification.lean

```diff
@@ -0,0 +1,233 @@
+/-
+Copyright (c) 2023 Dagur Asgeirsson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Dagur Asgeirsson
+-/
+import Mathlib.CategoryTheory.Adjunction.Opposites
+import Mathlib.CategoryTheory.Adjunction.FullyFaithful
+import Mathlib.CategoryTheory.Sites.Sheaf
+import Mathlib.CategoryTheory.Limits.Preserves.Finite
+/-!
+
+# Sheafification
+
+Given a site `(C, J)` we define a typeclass `HasSheaf J A` saying that the inclusion functor from
```

Comment:
This docstring has a typo and is not quite precise. The existence of a left adjoint is called `HasWeakSheafify` and the existence of sheafification is called `HasSheafify`.